.vsts-ci.yml

```
@@ -22,6 +22,10 @@ parameters:
   displayName: Run A Test Build
   type: boolean
   default: false
+- name: enableArm64Job
```

Maybe being nitpicky, but perhaps wouldn't hurt to specify it is Mac Arm64??
```suggestion
- name: enableMacArm64Job
```

---

As it stands, this is a temporary fix. So, I don't foresee this parameter staying in the code after we actually determine the problem with the job and fix it.

---

Oh, I can clarify that this is our only true Arm64 job. The Linux one only builds for Arm64, but on an X64 machine. And that job doesn't run tests either. This Mac Arm64 job is building for Arm64 on Arm64 and running tests on Arm64.